src/app/pages/PerpetualPage/hooks/usePerpetual_depositMarginToken.ts

```diff
@@ -39,6 +32,9 @@ export const usePerpetual_depositMarginToken = (
       nonce?: number,
       customData?: PerpetualTx,
     ) => {
+      const perpetualId = PerpetualPairDictionary.get(
```

Comment:
Use `currentPairType` (`selectPerpetualPage` selector) as a fallback. Do this for all the hooks.

Reply:
Also, we set `asset` in `send` to `Asset.BTCS` so retrieve just the pair (and rename `perpetualId` to `pair`) and use `pair.id` instead of `perpetualId` and `pair.collateralAsset` instead of `Asset.BTCS`.

Do this for all the hooks pls.